tree.go

```diff
@@ -1503,10 +1589,23 @@ func (n *LeafNode) GetProofItems(keys keylist, _ NodeResolverFn) (*ProofElements
 		}
 
 		esses []byte = nil // list of extension statuses
-		poass []Stem       // list of proof-of-absence stems
+		poass []Stem       // list of proof-of-absence and proof-of-expiry stems
 	)
 
-	// Initialize the top-level polynomial with 1 + stem + C1 + C2
+	// If the leaf node is expired, generate the proof of expiry.
+	if IsExpired(n.lastTs, curTs) {
```

Comment:
I don't think we need to change the API for `GetProofItems` to have a timestamp/epoch: this is because when we do the proof, the relevant elements of the tree have already been loaded with `Get`, which is where the detection can happen. It would be simpler to catch things there, and assume that, at proof creation time, if a node is not expired, then it's not expired in the db either.